pulp_rpm/tests/functional/api/test_crud_content_unit.py

```diff
@@ -94,19 +90,71 @@ def test_03_fully_update(self):
 
         This HTTP method is not supported and a HTTP exception is expected.
         """
-        attrs = _gen_content_unit_attrs(self.artifact)
+        attrs = _gen_rpm_package_attrs(self.artifact)
         with self.assertRaises(HTTPError):
             self.client.put(self.content_unit['_href'], attrs)
 
 
-def _gen_content_unit_attrs(artifact):
+def _gen_rpm_package_attrs(artifact):
     """Generate a dict with content unit attributes.
 
     :param: artifact: A dict of info about the artifact.
     :returns: A semi-random dict for use in creating a content unit.
     """
-    # FIXME: add content specific metadata here
-    return {'artifact': artifact['_href']}
+    name = choice(('bear', 'gorilla', 'kangaroo', 'dog'))
```

Comment:
Why not re-use the data from the the RPM used to create the `artifact`.

Reply:
My understanding is that it's supposed to be semi-random for the sake of the update test cases.  If it's not a little random, the update test cases won't be testing anything properly.

That's just an assumption based on the docstring though, I haven't looked through the history for rationale or anything.

If we were to just use the data from the RPM used to create the artifact, then we would either have to parse the RPM file to get that info (non-trivial), or we could hardcode it, in which case there's not much point in having a function for it since it could just be a dict in ```constants.py```